pykickstart/handlers/rhel8.py

```diff
@@ -0,0 +1,119 @@
+        "clearpart": commands.clearpart.F28_ClearPart,
+        "cmdline": commands.displaymode.F26_DisplayMode,
+        "device": commands.device.F24_Device,
+        "deviceprobe": commands.deviceprobe.FC3_DeviceProbe,
+        "dmraid": commands.dmraid.F24_DmRaid,
+        "driverdisk": commands.driverdisk.F14_DriverDisk,
+        "eula": commands.eula.F20_Eula,
+        "fcoe": commands.fcoe.RHEL8_Fcoe,
+        "firewall": commands.firewall.F28_Firewall,
+        "firstboot": commands.firstboot.FC3_Firstboot,
+        "graphical": commands.displaymode.F26_DisplayMode,
+        "group": commands.group.F12_Group,
+        "halt": commands.reboot.F23_Reboot,
+        "harddrive": commands.harddrive.FC3_HardDrive,
+        "hmc" : commands.hmc.F28_Hmc,
+        "ignoredisk": commands.ignoredisk.F14_IgnoreDisk,
+        "install": commands.install.F20_Install,
+        "iscsi": commands.iscsi.F17_Iscsi,
+        "iscsiname": commands.iscsiname.FC6_IscsiName,
+        "keyboard": commands.keyboard.F18_Keyboard,
+        "lang": commands.lang.F19_Lang,
+        "liveimg": commands.liveimg.F19_Liveimg,
+        "logging": commands.logging.FC6_Logging,
+        "logvol": commands.logvol.RHEL8_LogVol,
+        "mediacheck": commands.mediacheck.FC4_MediaCheck,
+        "method": commands.method.F28_Method,
+        "mount": commands.mount.F27_Mount,
+        "multipath": commands.multipath.F24_MultiPath,
+        "network": commands.network.F27_Network,
+        "nfs": commands.nfs.FC6_NFS,
+        "ostreesetup": commands.ostreesetup.RHEL8_OSTreeSetup,
+        "part": commands.partition.RHEL8_Partition,
+        "partition": commands.partition.RHEL8_Partition,
+        "poweroff": commands.reboot.F23_Reboot,
+        "raid": commands.raid.RHEL8_Raid,
+        "realm": commands.realm.F19_Realm,
+        "reboot": commands.reboot.F23_Reboot,
+        "repo": commands.repo.F27_Repo,
+        "reqpart": commands.reqpart.F23_ReqPart,
+        "rescue": commands.rescue.F10_Rescue,
+        "rootpw": commands.rootpw.F18_RootPw,
+        "selinux": commands.selinux.FC3_SELinux,
+        "services": commands.services.FC6_Services,
+        "shutdown": commands.reboot.F23_Reboot,
+        "skipx": commands.skipx.FC3_SkipX,
+        "snapshot": commands.snapshot.F26_Snapshot,
+        "sshpw": commands.sshpw.F24_SshPw,
+        "sshkey": commands.sshkey.F22_SshKey,
+        "text": commands.displaymode.F26_DisplayMode,
+        "timezone": commands.timezone.F25_Timezone,
+        "updates": commands.updates.F7_Updates,
+        "upgrade": commands.upgrade.F20_Upgrade,
+        "url": commands.url.F27_Url,
+        "user": commands.user.F24_User,
+        "vnc": commands.vnc.F9_Vnc,
+        "volgroup": commands.volgroup.RHEL8_VolGroup,
+        "xconfig": commands.xconfig.F14_XConfig,
+        "zerombr": commands.zerombr.F9_ZeroMbr,
+        "zfcp": commands.zfcp.F14_ZFCP,
+    }
+
+    dataMap = {
+        "BTRFSData": commands.btrfs.F23_BTRFSData,
```

Comment:
Should this line stay here?

Reply:
I don't think we've ever had a case where we removed a command that had associated data classes, so we're in uncharted territory here.  I don't really know what the right thing to do is, but I lean towards leaving it.  We'll still recognize the "btrfs" command for purposes of not breaking everyone's stuff, so we should probably leave the data class around for the same reason.  We should probably also just have some way of marking it as deprecated.

Reply:
Obviously in this patch, it should stay - this is just adding the structure to be modified later.

Reply:
Ok.